controllers/api/forum-routes.js

```diff
@@ -5,71 +5,75 @@ const withAuth = require('../../utils/auth');
 
 //gets all forum posts
 router.get('/', async (req, res) => {
-  console.log(req.session);
-    const { user } = req.session;
+  const { user } = req.session;
 
```

Comment:
This is not needed with `withAuth`, also `user` is a `req.session` object, which has a property `'loggedIn' = boolean`.
When you do a check for `user` you are checking if a session exists, when I think you are trying to check if a user is logged in. Which you can actually access with `req.session.loggedIn`. This btw is already handled by `withAuth`.